.github/workflows/linux.yml

```diff
@@ -22,3 +22,72 @@ jobs:
         with:
           name: ooniprobe-linux-amd64
           path: ./CLI/linux/amd64/ooniprobe
+
```

Comment:
I would be grateful if you could move this code to the `debian.yml` workflow :-). This is the workflow that we use for building for Debian, and I find it surprising that we build for Debian here. (I know we have too many workflows and so it's not immediately easy to understand which one to use, sorry!)